### ISSUE TITLE
Conflict with `spatie/laravel-medialibrary`

### ISSUE THREAD

**Issue by ewilan-riviere** [nunomaduro/larastan#1116]
- Larastan Version: 2.0.0
- `--level` used: 5
- Pull request with failing test:
- OS: Windows 10

### Description

When I install [`spatie/laravel-medialibrary`](https://github.com/spatie/laravel-medialibrary) *v10*, I've an error when I execute larastan. This error can be checked on [this repository](https://gitlab.com/ewilan-riviere/laravel-larastan).

```bash
composer require "spatie/laravel-medialibrary:^10.0.0"
```

### Laravel code where the issue was found

```bash
./vendor/bin/phpstan analyse
```

```bash
Note: Using configuration file C:\workspace\projets\laravel-app\phpstan.neon.
This file is included multiple times:
- C:\workspace\projets\laravel-app\vendor\nunomaduro\larastan\extension.neon

It can lead to unexpected results. If you're using phpstan/extension-installer, make sure you have removed corresponding neon files from your project config file.
```

Larastan works when I remove *laravel-media-library*.

**Comment by canvural** [comment]
Hi,

They require `phpstan/extension-installer` in their [`composer.json`](https://github.com/spatie/laravel-medialibrary/blob/main/composer.json#L38) Which is a tool that automatically registers an PHPStan extensions.

So please open an issue there. They should remove that from their composer.json

**Comment by ewilan-riviere** [comment]
Thanks!

**Comment by ewilan-riviere** [comment]
Fixed in [laravel-medialibrary 10.0.4](https://github.com/spatie/laravel-medialibrary/releases/tag/10.0.4).
